src/nl/hannahsten/texifyidea/inspections/latex/LatexCollapseCiteInspection.kt

```diff
@@ -114,7 +120,9 @@ open class LatexCollapseCiteInspection : TexifyInspectionBase() {
     }
 
     /**
-     * @author Hannah Schellekens
+     * @param citeBundle a bundle of cite commands that have to be merged into one command. All commands
+     * in this bundle have only required parameters, as cites with optional parameters should not
+     * be collapsed.
      */
     private inner class InspectionFix(val citeBundle: List<LatexCommands>) : LocalQuickFix {
```

Comment:
Use `@param` for type parameters, `@property` for property documentation and `@constructor` for the primary constructor. [docs](https://kotlinlang.org/docs/reference/kotlin-doc.html).

Reply:
I had no idea that `@property` and `@constructor` existed. Good to know :) I agree that following the docs is generally a good idea, however when using `@property` the property doesn't get rendered at all in the documentation popup, which is rather inconvenient.